server/src/main/java/org/apache/druid/segment/loading/StorageLocation.java

```diff
@@ -44,19 +45,25 @@
   private static final Logger log = new Logger(StorageLocation.class);
 
   private final File path;
-  private final long maxSize; // in bytes
+  private final long maxSizeBytes;
   private final long freeSpaceToKeep;
 
-  // Set of files stored under the given path. All accesses must be synchronized with currSize.
+  /**
+   * Set of files stored under the given path. All accesses must be synchronized with currSizeBytes.
+   */
+  @GuardedBy("this")
   private final Set<File> files = new HashSet<>();
 
-  // Current total size of files in bytes. All accesses must be synchronized with files.
-  private long currSize = 0;
+  /**
+   * Current total size of files in bytes. All accesses must be synchronized with files.
```

Comment:
"All accesses must be synchronized with files." sentence is strange. It seems to me that it should be just removed, `GuardedBy` already says everything needed.

Reply:
Removed.